addons/stock/models/stock_move.py

```diff
@@ -1020,7 +1021,7 @@ def _action_confirm(self, merge=True, merge_into=False):
                 move.product_id, move.product_uom_qty, move.product_uom,
                 move.location_id, move.rule_id and move.rule_id.name or "/",
                 origin, move.company_id, values))
-        self.env['procurement.group'].run(procurement_requests)
+        self.env['procurement.group'].run(procurement_requests, raise_user_error=(not self.env.context.get('from_orderpoint')))
```

Comment:
the outer parens are useless
raise_user_error=not self.env.context.get('from_orderpoint')

Reply:
hééééé